src/main/sensors/barometer.h

```diff
@@ -31,6 +31,11 @@ typedef enum {
 #define BARO_SAMPLE_COUNT_MAX   48
 
 typedef struct barometerConfig_s {
+    uint8_t baro_bustype;
+    uint8_t baro_spi_device;
+    ioTag_t baro_spi_cs;
```

Comment:
I prefer `baro_spi_cs`. Currently a mixture of `cs` and `csn` is used. I'd like to try and standardise on `csn` throughout.